rules/license-header.test.ts

```diff
@@ -5,14 +5,19 @@ import { RuleTester } from "@typescript-eslint/rule-tester";
 import { TSESLint } from "@typescript-eslint/utils";
 
 // should be the same of LICENSE_HEADER defined on license-header.js file.
+const LICENSE_TYPE = "MPL-2.0";
+const noLicense = "";
```

Comment:
Use the same style of variables, or camel case or upper snake case, don't mix them

Reply:
Fixed.